packages/pwa-module/Helper/WebpackConfig.php

```diff
@@ -59,7 +76,7 @@ class WebpackConfig implements \JsonSerializable
     const DEFAULT_SERVICE_WORKER_NAME = 'sw.js';
 
     /**
-     * @var \Magento\Framework\View\Config
+     * @var Config
      */
     private $_viewConfig;
```

Comment:
We shouldn't use `_` to indicate protected or private visibility. _Appreciate this was existing, but nice to fix in here_

https://www.php-fig.org/psr/psr-2/#42-properties

Reply:
Completely agree. Will improve PR soon.

Reply:
https://github.com/magento-research/pwa-studio/pull/105/commits/bbece38eec039ca0bab1591cc70e372f4d241e63